### PR TITLE
Flip kubemark canary job to prow

### PR DESCRIPTION
**Pull request by krzyzacy**
DinD for kubemark-5-prow-canary still fails with non-kubemark issue - I suspect that's a project issue because I'm pointing the run to my own project  - flip around the other canary which is currently green to prow and investigate from there.

/area jobs
/assign @shyamjvs

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -19287,7 +19287,7 @@ periodics:
         defaultMode: 256
         secretName: ssh-key-secret
 
-- name: ci-kubernetes-kubemark-5-prow-canary
```
Comment (shyamjvs) [inline prow/config.yaml]: Why are you removing the 5-node canary job?

Comment (krzyzacy) [reply prow/config.yaml]: the job was never configured properly though

File: jobs/config.json [hunk 2]
```diff
@@ -9309,27 +9307,6 @@
       "sig-scalability"
     ]
   },
-  "ci-kubernetes-kubemark-5-prow-canary": {
```
Comment (shyamjvs) [inline jobs/config.json]: Instead of removing the job, could you make its config match the regular kubemark-5 job?

File: jobs/config.json [hunk 3]
```diff
@@ -9309,27 +9307,6 @@
...
-      "--env-file=jobs/env/ci-kubernetes-kubemark-5-gce.env",
-      "--extract=ci/latest",
-      "--extract-source",
-      "--gcp-project=sen-lu-test",
-      "--gcp-zone=us-central1-f",
-      "--kubemark",
-      "--kubemark-nodes=5",
-      "--provider=gce",
-      "--test=false",
-      "--test_args=--ginkgo.focus=\\[Feature:Performance\\] --gather-resource-usage=true --gather-metrics-at-teardown=true --output-print-type=json",
```
Comment (shyamjvs) [inline jobs/config.json]: I guess this should be [Feature:Empty]

File: jobs/config.json [hunk 4]
```diff
@@ -9309,27 +9307,6 @@
...
     ]
   },
-  "ci-kubernetes-kubemark-5-prow-canary": {
-    "args": [
-      "--cluster=kubemark-5-prow-canary",
-      "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/env/ci-kubernetes-kubemark-5-gce.env",
-      "--extract=ci/latest",
-      "--extract-source",
-      "--gcp-project=sen-lu-test",
```
Comment (shyamjvs) [inline jobs/config.json]: Try changing this project to 'k8s-jenkins-kubemark'?

**Comment by krzyzacy** [comment]
@shyamjvs pointed to the same project as the other kubemark-5 job

File: jobs/config.json [hunk 5]
```diff
@@ -9315,14 +9315,13 @@
       "--env-file=jobs/platform/gce.env",
       "--env-file=jobs/env/ci-kubernetes-kubemark-5-gce.env",
       "--extract=ci/latest",
-      "--extract-source",
-      "--gcp-project=sen-lu-test",
+      "--gcp-project=k8s-jenkins-kubemark",
```
Comment (shyamjvs) [inline jobs/config.json]: Also I think there is `--gcp-node-image=gci` we're passing to the kubemark jobs.

Comment (shyamjvs) [reply jobs/config.json]: That might be the reason why those runs are failing during the ssh step where we're running some setup scripts (which iirc have gci-specific stuff that don't work on container_vm)

Comment (krzyzacy) [reply jobs/config.json]: that should be default now

Comment (krzyzacy) [reply jobs/config.json]: just in case, I added `--gcp-node-image=gci`

Comment (shyamjvs) [reply jobs/config.json]: I see. We probably want to get rid of the flag for the jobs then (though it can happen in a separate PR).

**Comment by shyamjvs** [comment]
/lgtm
Let me know how it goes. If it works, we can move the rest to cron too.

**Comment by krzyzacy** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-kubemark-5-prow-canary/818 is definitely passing now - I think the issue was my project :-(

**Pull request merged**